modules/1_opencv/src/opencv_coins.cpp

```diff
@@ -1,14 +1,36 @@
 #include "opencv_coins.hpp"
 
+
```

Comment:
Please, remove redundant strings

Reply:
It doesn't help

Reply:
Thanks. Problem was in last newline and string"CV Error"